src/include/sof/audio/component.h

```diff
@@ -140,6 +140,12 @@
 #define COMP_CMD_IPC_MMAP_VOL(chan)	(216 + chan)	/**< Volume */
 /** @}*/
 
+/** \name Register component macro
+ *  Must be used before void sys_comp_*_init(void){}
+ *  @{
+ */
+#define REGISTER_COMPONENT	__attribute__((section(".comp_init")))
```

Comment:
Can we call this DECLARE_COMPONENT since it's more closer to the usage.

Reply:
Sure, done